bitwise/src/main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <Ark/Module.hpp>
+#include <bitset>
+
+namespace ArkBitwise
+{
+    Value rshift(std::vector<Value>& n, Ark::VM* vm)
+    {
+
+        if (n.size() != 2)
+            throw std::runtime_error("bitwise:rshift: needs 1 argument: number");
+        if (n[0].valueType() != ValueType::Number 
+            && n[1].valueType() != ValueType::Number)
+            throw Ark::TypeError("bitwise:rshift the 2 arguments must be numbers"); 
+
+        int num = static_cast<int>(n[0].number());
+        int shift = static_cast<int>(n[1].number());
+        int shifted = num >> shift;
```

Comment:
Use long instead of int, since integers are on 32 bits, and our internal number type is double (64 bits)

Same thing for the other methods